### PR TITLE
sshfs 2.8

### PR DESCRIPTION
**Pull request by benesch**
with a fix for #16. I've also submitted the patch upstream: libfuse/sshfs#27.

### CODE REVIEW

File: Formula/sshfs.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Sshfs < Formula
   desc "File system client based on SSH File Transfer Protocol"
   homepage "https://osxfuse.github.io/"
-  url "https://github.com/osxfuse/sshfs/archive/osxfuse-sshfs-2.5.0.tar.gz"
-  sha256 "8ea4d3d5bc0f343998009d7eb138e3804490f6a22e890c6de4eadc6fd2414ae0"
+  url "https://github.com/libfuse/sshfs/releases/download/sshfs_2.8/sshfs-2.8.tar.gz"
```
Comment (DomT4) [inline Formula/sshfs.rb]: Why the URL move here? The original at https://github.com/osxfuse/sshfs still seems to be present?

Comment (ilovezfs) [inline Formula/sshfs.rb]: These appear to be two separate projects.

Comment (benesch) [inline Formula/sshfs.rb]: They've merged! See: https://github.com/libfuse/sshfs/pull/5. As libfuse/sshfs now supports OS X, osxfuse/sshfs is dormant.

Comment (DomT4) [inline Formula/sshfs.rb]: Thanks for the update here, and apologies for the delay in replying. Too many notifications 🙈. The URL change seems fine.

File: Formula/sshfs.rb [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+  sha256 "7f689174d02e6b7e2631306fda4fb8e6b4483102d1bce82b3cdafba33369ad22"
 
   option "without-sshnodelay", "Don't compile NODELAY workaround for ssh"
 
   depends_on "pkg-config" => :build
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "gettext" => :linked
```
Comment (DomT4) [inline Formula/sshfs.rb]: Can remove this line now; we've stopped using this syntax pending a deeper review into whether it's beneficial enough to us to counter the cost to contributors in having to add it all over the place.

File: Formula/sshfs.rb [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
   depends_on :osxfuse
   depends_on "glib"
 
-  # Fixes issue https://github.com/osxfuse/sshfs/pull/4
+  # Fixes issue https://github.com/osxfuse/sshfs/issues/17
```
Comment (DomT4) [inline Formula/sshfs.rb]: Looks like this should be https://github.com/libfuse/sshfs/pull/27?

File: Formula/sshfs.rb [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
   patch :DATA
```
Comment (DomT4) [inline Formula/sshfs.rb]: Any chance you can use the upstream patch URL now they've merged it, if that applies cleanly?

```ruby
patch do
  url "https://github.com/libfuse/sshfs/commit/e5acfce8eda218d.patch"
  sha256 "<mysha256>"
end
```

**Comment by benesch** [comment]
No worries about the delay! Thanks for the review. How's this look?

**Comment by DomT4** [comment]
Looks great to me. Merged in https://github.com/Homebrew/homebrew-fuse/commit/186077abd11ef068668087a687f068838df48aa3.

Thank you for your contribution to Homebrew @benesch, and your patience on getting this done. Appreciate both.

**Comment by benesch** [comment]
My pleasure. Thanks for merging!

**Pull request merged**